src/types/Filter.ts

```diff
@@ -0,0 +1,5 @@
+export enum Filter {
```

Comment:
pls see [enum naming conventio](https://michaelcurrin.github.io/dev-cheatsheets/cheatsheets/typescript/data-types/enums.html#:~:text=Naming%20convention,enum%20tend%20to%20be%20titlecase)

Reply:
FilterTodos would be a better name?